.github/workflows/deploy-to-staging.yml

```diff
@@ -30,3 +30,12 @@ jobs:
         BRANCH: gh-pages
         FOLDER: _site
         CLEAN: true
+    - name: Waiting 4 minutes for staging data deployment
+      run: |
+        sleep 240
+    - name: Rebuild site
+      run: |
+        curl -X POST https://api.github.com/repos/<staging org>/<staging site repo>/dispatches \
```

Comment:
@brockfanning this isn't quite ready - wondering what best way to get <staging org> and <staging site repo>

Reply:
Nevermind, I'll do it the same way you have done it in deploy-to-production